### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. Added Home Page of e-commerce

image

image

image

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
UI looks good and elegant, good to merge from side

File: index.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+                            <img class="card-image" alt="card-image"
+                                src="/assets/alchemist.jpg">
+                            <div class="card-text card6-text">
+                                <header>
+                                    <h3>Atomic Habits</h3><button class="rate-btn"><span>&#9733</span>4.5</button>
+                                </header>
+                                <div class="card-info">
+                                    <p>Juvita kon</p>
+                                    <div class="price">Rs. 300
+                                        <span class="price-old">RS. 600</span>
```
Comment (Adarsh-Patel-Dev) [inline index.html]: Just a suggestion 
use &#8377  for rupee symbol instead of Rs

File: index.html [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+                                    </div>
+                                </div>
+                            </div>
+            
+                            <footer>
+                                <p>Add to cart</p>
```
Comment (Adarsh-Patel-Dev) [inline index.html]: try to use some styling in <p> tag
it doesn't look like button

File: index.html [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+                                </header>
+                                <div class="card-info">
+                                    <p>Juvita kon</p>
+                                    <div class="price">Rs. 300
+                                        <span class="price-old">RS. 600</span>
+                                    </div>
+                                </div>
+                            </div>
+            
+                            <footer>
```
Comment (Adarsh-Patel-Dev) [inline index.html:129]: Why you are using footer tag here?

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (JuvitaSaini09) [reply index.html]: thanks for letting me know ,i have updated it

File: index.html (hunk 1 above)
Comment (JuvitaSaini09) [reply index.html]: I have made the change,thankyou for review

File: index.html (hunk 3 above)
Comment (JuvitaSaini09) [reply index.html:129]: i used it as cards footer

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**